cloudify_ansible/tasks.py

```diff
@@ -12,107 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from cloudify.decorators import operation
```

Comment:
@EarthmanT Do we still need to use operation ?

Reply:
@mabuaisha there have been discussions in R&D about making it mandatory again.

Reply:
Ok